### DISCUSSION TITLE
CucumberJS and Allure-CucumberJS: Result File Not Generated

### DISCUSSION THREAD

**Discussion by CoffeyTim** [allure-framework/allure2#2221] [Questions & Support] [answered]
Hello...I have done my due diligence searching for an answer to this, but I cannot figure it out. 

I am using "@cucumber/cucumber": "^8.0.0" to run my tests. 

- Example script in package.json:  "debug": "PWDEBUG=console cucumber-js --tags @focus --exit --profile"

I am using Allure-CucumberJS to try to generate a report.

- "allure-commandline": "^2.24.1"
- "allure-cucumberjs": "^2.9.2"
- "allure-js-commons": "^2.9.2"

In my cucumber.js I have the following:
`module.exports = {
  default: {
    ...common,
    format: 'src/reporter.js',
    paths: [
      'src/reporter.js'
    ]
  }`

My reporter.js contains the following:
`import { AllureRuntime, CucumberJSAllureFormatter } from "allure-cucumberjs";

export default class Reporter extends CucumberJSAllureFormatter {
  constructor(options) {
    super(options, new AllureRuntime({ resultsDir: "./allure-results" }), {});
  }
}`

To generate and open the Allure report I am running the following:

- allure generate allure-results --clean
- allure open ./allure-report/

The only file being generated in allure-results is the report.json, which contains the correct and updated info about the run.

So when I open the report it is empty.

**Comment by baev** [comment, marked as answer]
You can generate a working example using [Allure Start](https://allurereport.org/start/). As for today, it uses cucumber `^8.4.0` and has no issues

**Reply by CoffeyTim** [reply]
Thank you!
